### ISSUE TITLE
Feature Request: Keep unmatched images in the stream

### ISSUE THREAD

**Issue by MikeCostello** [mahnunchik/gulp-responsive#13]
It would be nice to let all `src` files pass through when `strictMatchImages: false`.

Currently I split/concat the stream.

**Comment by mahnunchik** [comment]
Hi @MikeCostello 

If I understand your question correctly, it is possible to pass files through by using the following configuration:
```js
gulp.task('images', function () {
  return gulp.src('src/**/*{png,jpg}')
    .pipe($.responsive({
      '**/*': {}
    }))
    .pipe(gulp.dest('dist'));
});
```
This glob pattern matches all files so it is configuration for all files.

More useful example. It strips metadata and sets quality:
```js
gulp.task('images', function () {
  return gulp.src('src/**/*{png,jpg}')
    .pipe($.responsive({
      '**/*': {}
    },{
      withMetadata: false,
      quality: 90
    }))
    .pipe(gulp.dest('dist'));
});
```

**Comment by MikeCostello** [comment]
The `**/*` glob works as the last item, letting earlier configs be applied and still maintaining the all of the files from the input stream. Should this be the default behavior, with the addition that files not matched are piped through untouched?

**Comment by mahnunchik** [comment]
Hi @MikeCostello 

Landed in the 1.1.0 release
